ssas/okta/okta.go

```diff
@@ -35,26 +45,38 @@ func config() error {
 	if at != "" {
 		at = "[Redacted]"
 	}
-
 	OktaAuthString = fmt.Sprintf("SSWS %s", oktaToken)
+	OktaBaseUrl = os.Getenv("OKTA_CLIENT_ORGURL")
```

Comment:
```suggestion
	OktaBaseURL = os.Getenv("OKTA_CLIENT_ORGURL")
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms